examples/text-classification/README.md

```diff
@@ -83,6 +83,23 @@ After a few hours of training, your network loss and accuracy may look like:
 
 ![loss](dbpedia-loss.png)
 
+**NOTE**: There is a minor bug in cuDNN v5.1.10 that you may run into with this example.
+On Kepler GPUs (e.g. K80), you may have very bad performance (time estimates in the hundreds of days!).
+If so, cuDNN is suggesting an inappropriate algorithm to use.
+You can override this easily - simply replace this line:
+```lua
+    -- [(1018-3)/3+1=339] x 256
+    net:add(backend.TemporalConvolution(256, 256, 7))
+```
+With these lines:
+```lua
+     -- [(1018-3)/3+1=339] x 256
+    local conv2 = backend.TemporalConvolution(256, 256, 7)
+    conv2:setMode(nil, 0, nil)
```

Comment:
`setMode()` is undefined for the `nn` backend (and calling it would yield an exception) but since this note specifically states that this patch applies to CuDNN v.5 then I believe it's OK.